AdyenActions/UI/View Controllers/Voucher/VoucherView.swift

```diff
@@ -247,16 +247,17 @@ internal final class VoucherView: UIView, Localizable {
         )
     }
     
-    override public func didMoveToWindow() {
-        super.didMoveToWindow()
+    override public func didMoveToSuperview() {
+        super.didMoveToSuperview()
         updateLogo()
     }
     
     private func updateLogo() {
-        if window != nil {
-            imageLoadingTask = logo.load(url: model.logoUrl, using: model.imageLoader)
-        } else {
+        if superview == nil {
+            imageLoadingTask?.cancel()
```

Comment:
Maybe you want to do that in the didSet of the imageLoadingTask? So the previous task always gets cancelled first

Reply:
Good point, will do!

Reply:
Good catch btw - was an oversight on my side 🙈